typescript/sdk/src/index.ts

```diff
@@ -6,6 +6,8 @@ export {
   BaseSealevelAdapter,
   MultiProtocolApp,
 } from './app/MultiProtocolApp.js';
+export { S3Config, S3Receipt, S3Wrapper } from './aws/s3.js';
```

Comment:
Tough to properly review the changes here with so much churn. I've asked this before: why is the index order shuffling so much when different devs change it? Why isn't prettier and/or the vscode ts import organizer action being consistent?